data/io.crow_translate.CrowTranslate.metainfo.xml

```diff
@@ -32,6 +32,8 @@
 
   <launchable type="desktop-id">io.crow_translate.CrowTranslate.desktop</launchable>
 
+  <translation type="qt">Crow Translate/Crow Translate/translations/crow-translate</translation>
```

Comment:
Just an explanation: according to the spec, this should expand into `$PREFIX/share/Crow Translate/Crow Translate/translations/crow-translate_$LANG.qm`, and if the `.qm` file exist for given `$LANG` - assume the app is translated to it.

Reply:
Noted, thanks!